cmd/auth/main.go

```diff
@@ -142,6 +146,11 @@ func loadConfig() config {
 		SSLRootCert: mainflux.Env(envDBSSLRootCert, defDBSSLRootCert),
 	}
 
+	loginDuration, err := time.ParseDuration(mainflux.Env(envLoginDuration, defLoginDuration))
```

Comment:
Instead of panic can we use defLoginDuration value 
And log the envLoginDuration as warning or error 
If the defLoginDuration value got error during parse , then log panic

Like 
```
loginDuration, err := time.ParseDuration(envLoginDuration)
if err != nil {
	logger.Info("Invalid Login Duration %v: %v", envLoginDuration, err)
	loginDuration , err := time.ParseDuration(defLoginDuration)
	if err != nil  {
               log.Painc(fmt.Sprintf("Invalid default login duration %v: %v", defLoginDuration err))
       }
}	
```

Reply:
That's is one way to do it. TBH, since `default` is not exactly `fallback` (despite the params naming in `mainflux.Env`), and due to code simplicity, I prefer the current solution. It's also aligned with the rest of the codebase.